lib/ui/pages/news_list/news_list_item.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:allthenews/domain/news.dart';
+import 'package:flutter/material.dart';
+
+class NewsListItem extends StatelessWidget {
+  final News news;
+
+  NewsListItem({Key key, @required this.news})
+      : assert(news != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        _buildImage(),
+        SizedBox(width: 20),
+        Expanded(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              _buildTitle(context),
+              SizedBox(height: 8),
+              _buildSubtitle(context),
+            ],
+          ),
+        )
+      ],
+    );
+  }
+
+  Widget _buildTitle(BuildContext context) {
+    return Text(
+      news.title,
+      style: _getTitleStyle(context),
+      maxLines: 2,
+      overflow: TextOverflow.ellipsis,
+    );
+  }
+
+  Widget _buildImage() {
+    return ClipRRect(
+      borderRadius: BorderRadius.circular(8.0),
+      child: Image.network(
+        'https://i.picsum.photos/id/9/800/800.jpg',
+        width: 110,
+        height: 90,
+        fit: BoxFit.fill,
+      ),
+    );
+  }
+
+  Widget _buildSubtitle(BuildContext context) {
+    final style = _getSubtitleStyle(context);
+
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(news.date, style: style),
+        Container(
+            alignment: Alignment.center,
+            width: 20,
+            child: Text(
+              String.fromCharCode(0x2022),
```

Comment:
to też by poszło do constów wtedy i miało nazwę, bo teraz trzeba widzieć UI, żeby wiedzieć co to jest